client/packages/system/src/Patient/CreatePatientModal/PatientFormTab.tsx

```diff
@@ -1,117 +1,57 @@
-import React, { FC, useMemo } from 'react';
+import React, { FC, useState } from 'react';
 import {
-  BaseDatePickerInput,
-  BufferedTextInput,
-  DateUtils,
-  InputWithLabelRow,
-  Select,
-  Stack,
-  useFormatDateTime,
-  useTranslation,
-} from '@openmsupply-client/common';
-import { Gender, usePatientCreateStore } from '@openmsupply-client/programs';
+  Gender,
+  JsonData,
+  JsonForm,
+  useFormSchema,
+  usePatientCreateStore,
+} from '@openmsupply-client/programs';
 import { PatientPanel } from './PatientPanel';
 
+type Patient = {
+  code?: string;
+  code2?: string;
+  firstName?: string;
+  lastName?: string;
+  dateOfBirth?: string;
+  gender?: Gender;
+};
+
 export const PatientFormTab: FC<PatientPanel> = ({ patient, value }) => {
   const { updatePatient } = usePatientCreateStore();
-  const t = useTranslation('patients');
-
-  const genderOptions = useMemo(
-    () =>
-      Object.keys(Gender).map(key => ({
-        label: key,
-        value: key,
-      })),
-    []
+  const { data: patientCreationUI, isError, isLoading } = useFormSchema.document.byType(
+    'PatientCreationJSONForm'
```

Comment:
Think it would be nice to build in a fallback, i.e. if the PatientCreationJSONForm is not configured and the returned data is undefined than use a very basic hardcoded default schema...

For the type I would call it `PatientCreationJSONForms` because the its JSONForms and not JSONForm...

Reply:
You mean a default form schema, or just displaying the component that was there before if the creation form doesn't exist? 🤔

Reply:
You could just have a hard coded json file for the default schemas in the UI code, think this would be the simplest solution(?)

Reply:
yes, I did get this initially. shouldn't be a problem for normal usage though.

<img width="589" alt="Screenshot 2023-03-02 at 3 57 23 PM" src="https://user-images.githubusercontent.com/9192912/222320363-8e620cb6-0993-436f-97ef-46a0a11483ad.png">